src/styles/_catalog.scss

```diff
@@ -1,22 +1,44 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: $gap;
+  max-width: 100%;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    display: grid;
+    grid-template-columns: repeat(2, $card-width);
+    justify-content: center;
+    gap: $gap;
+    max-width: 100%;
+  }
```

Comment:
you shoud change only grid-template-columns. 
the rest needs to be added once to the parent element

Reply:
ohhh... you're right! my mistake